### ISSUE TITLE
[AdaptiveCards- Designer - Help]: 2 closed button as 'X and close' is set in the 'Help' dialog box and working of both are same.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8174]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: [https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)

**Repro Steps:**
1. Open above URL with valid credential.
2. Navigate to 'Designer' link and press enter key to activate it.
3. Navigate to 'Help' button and press enter key.

**Actual Result:**
2 close buttons as 'X' and 'Close' are defined in the 'Help' dialog box, having the same functionality.

**Expected Result:**
Only one close button should be defined on the dialog box, and it should be descriptive so that user can easily understand about the element. Also, after activating the close button focus is lost.

**User Experience:**
It will impact all the user and mostly screen reader users, if the 2 close buttons are defined, which might be confusing.

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="952" alt="Usable - 2 close button is defined" src="https://user-images.githubusercontent.com/95913748/208674635-0922f852-c39d-46e6-bdf1-2f36a8e0766e.png">

### Card JSON

```JSON
NA
```
